src/com/seafile/seadroid2/ui/activity/AccountsActivity.java

```diff
@@ -235,9 +235,8 @@ public void onBackPressed() {
         Account account = accountManager.getCurrentAccount();
         if (account == null) {
             // force exit when current account was deleted
-            Intent i = new Intent();
-            i.setAction(Intent.ACTION_MAIN);
-            i.addCategory(Intent.CATEGORY_HOME);
+            Intent i = new Intent(this, BrowserActivity.class);
+            i.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
```

Comment:
This fix is not correct, reproduce the error as following:
1. Open Seafile App for Android
2. Go to Settings -> Sign out -> click Confirm button, then it navigates to Accounts list
3. Choose and click one Account, re-enter password -> Login, then it navigates to the LIBRARY tab
4. Browsing file, enter a folder in order to make some browsing history
5. Click HOME button, then click the Launcher icon
~~6. Restore the browsing progress (correct)~~
6. It shows the Account list (incorrect)